src/plugins/intel_gpu/src/graph/program_node.cpp

```diff
@@ -1475,7 +1491,7 @@ void program_node::init_onednn_primitive_attributes() {
                 dnnl::algorithm alg = onednn::convert_activation_func(fused_desc->activation_function);
                 if (alg == dnnl::algorithm::undef)
                     OPENVINO_THROW("Activations that are undef algorithms must be converted to other activations before "
-                                   "pushing to post-op.");
+                                    "pushing to post-op.");
```

Comment:
No need to change it. Please update it too?

Reply:
I do not modify it manually but I rollback it.